chart/chart.js

```diff
@@ -0,0 +1,41 @@
+const choiceResults = JSON.parse(localStorage.getItem('CHOICES'));
+
+const ids = [];
+const clicked = [];
+const seen = [];
+
+choiceResults.forEach((item) => {
+    if (item.timesClicked > 0) {
+        ids.push(item.name);
```

Comment:
you are not storing the name of the item in your array you are pulling from local storage.  if you push in item.id.  the chart works with the IDs as the labels